scripts/dts/gen_defines.py

```diff
@@ -242,6 +242,11 @@ def write_props(node):
         elif prop.type == "uint8-array":
             out_node_init(node, ident,
                           [f"0x{b:02x}" for b in prop.val])
+        elif prop.type == "phandle":
+            out_node(node, ident, "DT_" + node_ident(prop.val))
+        elif prop.type == "phandles":
+            for i, val in enumerate(prop.val):
+                out_node(node, f"{ident}_{i}", "DT_" + node_ident(val))
```

Comment:
A comment explaining that this is just generating a prefix would help here. Very cryptic if you don't know about `DT_PHANDLE_GET_PROP()`.

Reply:
Added some comments.

Reply:
Comments LGTM.